docs/README.md

```diff
@@ -19,8 +19,8 @@ Then, run [`packer init`](https://www.packer.io/docs/commands/init).
 packer {
   required_plugins {
     linode = {
-      version = ">= 1.0.0"
-      source  = "github.com/hashicorp/linode"
+      version = ">= 1.0.1"
+      source  = "github.com/hashicorp/packer-plugin-linode"
```

Comment:
The source reference is not the actual full URL. The common prefix `packer-plugin-` is automatically applied during the init process.
```suggestion
      source  = "github.com/hashicorp/linode"
```


 By including it in the source reference you are actually telling Packer to try and download the plugin named `packer-plugin-packer-plugin-linode`, which will result in the following error. 

```
Plugin source "github.com/hashicorp/linode" has a type with the prefix
"packer-plugin-", which isn't valid. Although that prefix is often used in the
names of version control repositories for Packer plugins, plugin source strings
should not include it.

Did you mean "github.com/hashicorp/linode"?
```

Reply:
Ah, good to know. Thank you for the detailed explanation!